web/site/js/vis/PixiVisualizer.js

```diff
@@ -235,7 +235,12 @@ export default class PixiVisualizer extends VObject {
         }
 
         let src = this.nodes[srcId];
-        if (dstId == -1) {
+
+        let frameType = mvInfo.getFrameControl() & 0x0007;
+        if (frameType === 2) {
```

Comment:
Better define enumerations for `Frame Type`.

Reply:
hardcoding removed